datasciencebr/rosie
# Refactor election expenses classifier

fixes #50 

This a **WIP** I just opening it to have feedbacks earlier.
 
 **What is the purpose of this Pull Request?**
The purpose of this PR is make the classifier [election_expenses_classifier.py](https://github.com/datasciencebr/rosie/blob/master/rosie/chamber_of_deputies/classifiers/election_expenses_classifier.py) easier to understand, including the tests.
 
 **What was done to achieve this purpose?**
I renamed some variable to be more meaningful and some refactoring.
 
 **How to test if it really works?**
Just run the Rosie test and see if everything keep working. A refactor should not change any behaviour or feature. `python rose.py test`
 
 **Who can help reviewing it?**
@anaschwendler @cuducos @jtemporal @Irio 
 
 **TODO**

- [x] Refactoring classifier source code
- [x] Give a more meaningful name for the subject under test
- [x] Refactor tests name to be more meaningful
- [x] Meaningful constant for `409-0 - CANDIDATO A CARGO POLITICO ELETIVO`
- [x] Avoid read from a file where it's not necessary to improve test performance
